src/extensions/default/CodeFolding/Prefs.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Wrapper around brackets pref system to ensure preferences are stored in in one single object instead of using multiple keys. 
+ * This is to make it easy for the user who edits their preferences file to easily manage the potentially numerous lines of preferences generated by the persisting code-folding state.
+ * @author Patrick Oladimeji
+ * @date 3/22/14 20:39:53 PM
+ */
+/*jslint vars: true, plusplus: true, devel: true, nomen: true, indent: 4, maxerr: 50 */
+/*global define, brackets*/
+define(function (require, exports, module) {
+    "use strict";
+    var PreferencesManager      = brackets.getModule("preferences/PreferencesManager"),
+        stateManager            = PreferencesManager.stateManager.getPrefixedSystem("code-folding"),
+		DefaultSettings			= require("DefaultSettings"),
+        store = {},
+		settings = {},
+        folds = "folds";
+    
+    function simplify(folds) {
+        if (!folds) { return folds; }
+        var res = {}, range;
+        Object.keys(folds).map(function (line) {
```

Comment:
Why call `map` when you are not really transforming data and creating a new array form it?  A forEach would suffice.